Flea/Shared/ReservationOverview.razor

```diff
@@ -0,0 +1,34 @@
+<div class="grid-container-res">
+    <div class="res-txt">@Name</div>
+    <div class="res-txt">@Stande</div>
+    <div class="res-txt">@(PaymentStatus ? "Betalt" : "Ej betalt")</div>
+    <button class="button res-del-button" @onclick="Delete">Slet</button>
+    <button class="button res-edit-button" @onclick="Edit">Rediger</button>
+</div>
+

```

Comment:
The file/component name is slightly misleading as it implies that component implements an overview of reservations, however this is in actuality done by the `Booking` view. This component merely an item in a overview reservation.

Reply:
This seems to have been solved in 446f8e0